packages/devtools_app/lib/src/screens/vm_developer/object_inspector_view_controller.dart

```diff
@@ -47,6 +50,8 @@ class ObjectInspectorViewController extends DisposableController
       programExplorerController
         ..initialize()
         ..initListeners();
+      // TODO(bkonyi):
+//      codeViewController.initialize();
```

Comment:
why is this commented out?

Reply:
Just old code, `CodeViewController` doesn't have an `initialize()` method.